src/components/SectioningContent/SectioningContent.stories.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { Story } from '@storybook/react'
+
+import { Heading } from '../Heading'
+
+import { Article, Aside, Nav, Section, SectioningFragment } from './SectioningContent'
+
+export default {
+  title: 'SectioningContent',
+  component: Section,
+  subcomponents: {
+    Article,
+    Aside,
+    Nav,
+    Section,
+    SectioningFragment,
+  },
+  parameters: {
+    withTheming: true,
+  },
+}
+
+export const SectioningContentStyle: Story = () => (
```

Comment:
- `Style` いる?
- `Story` は deprecated だったはず

```suggestion
export const SectioningContent: StoryFn = () => (
```